Shared/FeatureSwitch.swift

```diff
@@ -50,4 +56,14 @@ open class FeatureSwitch {
         let hash = abs(uuidString.hashValue)
         return hash % 100 < self.percentage
     }
+
+    open func enrollMember(_ prefs: Prefs) {
+        let key = "feature_switches/\(self.featureID)_enabled"
+        prefs.setBool(true, forKey: key)
+    }
+
+    open func removeMember(_ prefs: Prefs) {
+        let key = "feature_switches/\(self.featureID)_enabled"
+        prefs.setBool(false, forKey: key)
+    }
```

Comment:
Not sure I know the answer to this: should this be written as 

```swift
func setMembership(_ isEnabled: Bool, for pref: Prefs)
```

or something like it?